python/ql/test/experimental/meta/ConceptsTest.qll

```diff
@@ -570,6 +570,20 @@ class CsrfLocalProtectionSettingTest extends InlineExpectationsTest {
       if p.csrfEnabled()
       then tag = "CsrfLocalProtectionEnabled"
       else tag = "CsrfLocalProtectionDisabled"
+
+class XmlParsingTest extends InlineExpectationsTest {
+  XmlParsingTest() { this = "XmlParsingTest" }
```

Comment:
this bit is not going to work after the merge :flushed: my approval was just a second too fast :joy: